curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md

```diff
@@ -58,7 +58,7 @@ Initially, the `span` elements should contain the code `&#9825;` to represent an
 
 ```js
 const inputs = document.querySelectorAll('ul li span.favorite-icon');
-assert(inputs.length)
+assert.notEmpty(inputs)
```

Comment:
```suggestion
assert.isNotEmpty(inputs)
```
testing is recommented

Reply:
Dang it. I really should have stuck to one file at a time.